charts/matrixbot/templates/deployment.yaml

```diff
@@ -5,10 +5,18 @@ metadata:
   labels:
     app: {{ .Release.Name }}
 spec:
+  {{ if or (eq .Values.sqliteDB.enabled true) (eq .Values.sqliteDB.enabled true) }}
+  replicas: 1
```

Comment:
this is a bit ugly I know, the ideal solution would be to switch to a statefulset {{ if or (eq .Values.sqliteDB.enabled true) (eq .Values.sqliteDB.enabled true) }}